internal/handler/msg_listcollections.go

```diff
@@ -81,23 +81,47 @@ func (h *Handler) MsgListCollections(ctx context.Context, msg *wire.OpMsg) (*wir
 		d := must.NotFail(types.NewDocument(
 			"name", collection.Name,
 			"type", "collection",
+			"idIndex", must.NotFail(types.NewDocument(
+				"v", int32(2),
+				"key", must.NotFail(types.NewDocument("_id", int32(1))),
+				"name", "_id_",
+			)),
 		))
 
+		options := must.NotFail(types.NewDocument())
+		info := must.NotFail(types.NewDocument("readOnly", false))
+
+		if collection.Capped() {
+			// capped is not set when it's false as of MongoDB 7
```

Comment:
That kind of comment could be applied to 80% of FerretDB lines – we do things the same way as MongoDB. It is the opposite that is notable. But this is the same and also checked by tests, so it does not add anything. I think we should remove it.

Reply:
Removed!